devlab/tests/test_group_verification.py

```diff
@@ -86,8 +88,8 @@ def _prepare_files(self, grouping_by):
         with open(file_to_write_into, 'w') as stream:
             yaml.dump(self.pre_conf_dict, stream, default_flow_style=False)
         fab_path = os.path.join('devlab/tests', self.new_file_name)
-        cmd = 'cd {} && fab get_groups:configuration.ini,{}'.format(main_folder,
-                                                                    fab_path)
+        cmd = 'cd {} && fab get_groups:{}configuration.ini,' \
+              '{}'.format(main_folder, 'devlab/tests/', fab_path)
```

Comment:
Can not be done by ```os.path.join``` because ```config_ini``` element consists of two "paths" - ```configuration.ini``` file path and ```grouping.yaml``` file path which are splitted by ','. resulting command hat to be:
```'cd /home/vyerys/CloudFerry/CloudFerry && fab get_groups:devlab/tests/configuration.ini,devlab/tests/test_file.yaml'```